src/gleam/erlang/process.gleam

```diff
@@ -219,7 +218,11 @@ pub type ExitMessage {
 pub type ExitReason {
   Normal
   Killed
+  /// The process was killed using `send_abnormal_exit`
+  /// or by using Erlang's `exit` with a string value.
   Abnormal(reason: String)
+  /// The process died for some reason not covered by the other variants.
+  Unexpected(reason: Dynamic)
```

Comment:
Let's keep abnormal but change it to dynamic, seeing as unexpected isn't a term in OTP